rasa_sdk/interfaces.py

```diff
@@ -66,6 +68,7 @@ def __init__(
         self.latest_message = latest_message if latest_message else {}
         self.active_loop = active_loop
         self.latest_action_name = latest_action_name
+        self.stack = stack
```

Comment:
The ticket also mentions that we should place this access behind a feature flag, so I'd actually make this a private attribute and then implement a getter `@property` to access the stack or the current context only if the feature flag is active, otherwise log a warning or raise an exception, whatever is expected from a feature flag.

Reply:
I've asked for the decision made regarding this.

Reply:
yes exactly 👍